test/regress/regress1/datatypes/dt-color-2.6.smt2

```diff
@@ -11,7 +11,7 @@
 
 (assert (or (distinct a b c d)
  (< (match a ((red 5) (green 3) (blue 2))) 0)
- (< (match b ((red 2) (_ 1))) 0)
+ (< (match b ((red 2) (x 1))) 0)
```

Comment:
They are, but I found out our parser is non-compliant :)  We don't accept `_` as a variable name, whereas in the parser prior to this commit, `_` was hardcoded since I thought `_` was fixed.

Reply:
Oh... I can look into fixing the parser. Let me open an issue.

Reply:
Alright, it is actually not quite clear what the real answer is here but this is an issue for another PR.